neuralprophet/forecaster.py

```diff
@@ -811,6 +813,18 @@ def predict(self, df, decompose=True, raw=False):
             forecast = pd.concat((forecast, fcst), ignore_index=True)
         df = df_utils.return_df_in_original_format(forecast, received_ID_col, received_single_time_series)
         self.predict_steps = self.n_forecasts
+        # Conformal prediction interval with q
+        if self.q_hats:
+            if self.conformal_method == "naive":
+                df["yhat1 - qhat1"] = df["yhat1"] - self.q_hats[0]
+                df["yhat1 + qhat1"] = df["yhat1"] + self.q_hats[0]
+            else:  # self.conformal_method == "cqr"
+                quantile_hi = str(max(self.config_train.quantiles) * 100)
```

Comment:
This will skip all but the top and bottom quantiles - if a user specifies more than 2, they are not conformalized.
We should either raise an error in this case in the conformalize step (and here), or adopt the conformalize method for more than 2 quantiles. Or am I misreading something here?

Reply:
If the CP is "naive", then NP doesn't use the quantiles at all. If CP is "CQR", then it will take the largest and smallest value in the quantile, regardless of the length of quantiles list.

I can make an assert check that the quantiles list need to have at least 2 values, for high and low, but that can be done as a separate PR.